CHANGELOG.md

```diff
@@ -1,5 +1,174 @@
 [//]: # (current developments)
 
+## 24.9.0 (2024-09-26)
+
+### Enhancements
+
+* Add `conda.core.index.Index` as a faster drop-in replacement of the realized dictionary index. Note: The loggers are no longer implicitly initialized when fetching the index, instead explicitly call `conda.gateways.logging.initialize_logging`. (#13880, #14267)
+* Alias `conda env remove` command to `conda remove --all`. (#13977)
+* Add a new health check to `conda doctor` that detects if the `REQUESTS_CA_BUNDLE` env var points to a non-existent file. (#12905 via #14037)
+* Add `--sha256` flag to `conda list --explicit` so it lists URLs with a SHA256 hash instead of MD5 and make `conda install|create` compatible with these inputs. (#2903, #7882 via #14048)
+* Report `conda` version used to generate a `@EXPLICIT` text file. (#14048)
+* Update health checks outputs for consistency. (#14049 via #14079)
+* Fix Windows to Unix path conversion to handle UNC mounts and root paths. (#14157)
+* Add a new `denylist_channels` config option and `CONDA_DENYLIST_CHANNELS` environment variable to explicitly deny using specific channels globally, which is complementing the already existing `allowlist_channels` config option. (#14176 via #14196)
+* Add `conda commands` subcommand. (#14215)
+* Add `conda.testing.fixtures.session_capsys`. Use this to capture stdout and stderr within module, package, and session scoped fixtures. (#14243)
+* Add `conda.testing.fixtures.session_conda_cli`. Use this to invoke conda commands within module, package, and session scoped fixtures. (#14243)
+* Add `conda.testing.fixtures.session_tmp_env`. Use this to create a conda environment within module, package, and session scoped fixtures. (#14243)
+
+### Bug fixes
+
+* Prevent directories which contain conda environments from being specified as an environment prefix when creating new envs; this provides guardrails against the accidental deletion of environments via commands such as `conda rename`. (#13955)
+* Enable `conda doctor` to check whether the environment exists or not before trying to generate a health report. (#14042 via #14112)
+* Fix `conda.common.configuration.ObjectParameter`'s ability to appropriately handle defaults. (#14128 via #14149)
```

Comment:
This refers to the wrong issue, we want https://github.com/conda/conda/issues/14148 not https://github.com/conda/conda/issues/14128:

```suggestion
* Fix `conda.common.configuration.ObjectParameter`'s ability to appropriately handle defaults. (#14148 via #14149)
```